app/src/main/java/org/luxoft/sdl_core/BleCentralService.java

```diff
@@ -12,22 +12,25 @@
         public static final String TAG = BleCentralService.class.getSimpleName();
         public final static String ACTION_START_BLE = "ACTION_START_BLE";
         public final static String ACTION_STOP_BLE = "ACTION_STOP_BLE";
-        public final static String GET_CLIENT_MESSAGE = "GET_CLIENT_MESSAGE";
         public final static String ON_BLE_READY = "ON_BLE_READY";
-        public final static String ON_MOBILE_REQUEST_RECEIVED = "ON_MOBILE_REQUEST_RECEIVED";
+        public final static String ON_MOBILE_MESSAGE_RECEIVED = "ON_MOBILE_REQUEST_RECEIVED";
```

Comment:
```suggestion
        public final static String ON_MOBILE_MESSAGE_RECEIVED = "ON_MOBILE_MESSAGE_RECEIVED";
```

Reply:
Fixed in 8effe4942498195cce7a335540df658a92a3e296